lib/modules/Server.js

```diff
@@ -13,6 +13,31 @@ import actions from './actions';
 import { createQuery } from './query';
 import { METHODS } from './router';
 
+export const dispatchInitialShell = async (ctx, dispatch) => {
```

Comment:
Not sure if it belongs in the more generic "actions.js" instead, but since it's specifically for the server, I put it here. I could also put it in a serverActions.js or something instead.